applications/asset_tracker/src/cloud_codec/cloud_codec.c

```diff
@@ -52,184 +61,165 @@ struct cloud_sensor_chan_cfg {
 	struct sensor_chan_cfg cfg;
 };
 
-#define CMD_ARRAY(...) ((struct cmd[]) {__VA_ARGS__})
+#define CMD_ARRAY(...) ((struct cmd[]){ __VA_ARGS__ })
 
-#define CMD_NEW_TYPE(_name, _type)			\
-	{ 						\
-		.name = STRINGIFY(_name),		\
-		.type = _type, 				\
+#define CMD_NEW_TYPE(_type)                                                    \
```

Comment:
This one seems two spaces behind of others.

Reply:
this is what clang-format gives.  same below.

Reply:
here is a diff, uncrustify output on the left, clang-format on the right.
which do i use?
![merge](https://user-images.githubusercontent.com/4138391/70261968-491ddf00-1748-11ea-8e07-291c8d574ca4.png)

Reply:
Sorry for late response. The more I see quirks like these the less confident I am about using these formatters. IMO neither of the two above is correct. The kernel coding guide seems to prefer to align the backslashes:
https://www.kernel.org/doc/html/latest/process/coding-style.html#macros-enums-and-rtl
IMO, you should either align (but truly align every single line), or not align at all. Anything else simply looks odd.

Reply:
updated with no alignment.